content/docs/csidriver/installation/operator/powerflex.md

```diff
@@ -39,7 +39,7 @@ Kubernetes Operators make it easy to deploy and manage the entire lifecycle of c
           password: <password in base64>
 ```  
   - Create secret for FTP side by using the command `kubectl create -f sdc-repo-secret.yaml`.
-- Optionally, enable sdc monitor by uncommenting the section for sidecar in manifest yaml.
+- Optionally, enable sdc monitor by uncommenting the section for sidecar in manifest yaml. Please note that you will need to edit the value fields under the HOST_PID and MDM fields by filling the empty quotes with host PID and the MDM IPs. Not filling in these values will cause an error, so please leave this commented out if you don't intend on using it.
```

Comment:
If it is required to be filled in, then it really should not be listed as "Optionally...."

Reply:
It's required if you're using the sidecar. The "optionally" refers to the sidecar itself

Reply:
Thanks @jooseppi-luna! Is there a way to bold the "Please note..." or "Note:..."

Reply:
@AronAtDell thanks for the suggestion! Let me know how the clarification I submitted looks